sycl/test-e2e/Graph/Threading/submit.cpp

```diff
@@ -44,12 +49,19 @@ int main() {
   run_kernels_usm(Queue, Size, PtrA, PtrB, PtrC);
   Graph.end_recording();
 
+  std::vector<exp_ext::command_graph<exp_ext::graph_state::executable>>
+      GraphExecs;
+  for (unsigned i = 0; i < NumThreads; ++i) {
+    GraphExecs.push_back(Graph.finalize());
+  }
+
   Barrier SyncPoint{NumThreads};
 
   auto FinalizeGraph = [&](int ThreadNum) {
```

Comment:
```suggestion
  auto SubmitGraph = [&](int ThreadNum) {
```
Suggest renaming since this function no longer finalizes but submits instead.